app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,26 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
great job adding your cdn assets to a partial! This helps keep the app layout file clean :)